test/terraform_aws_encrypted_test.go

```diff
@@ -0,0 +1,83 @@
+// =================================================================
+//
+// Work of the U.S. Department of Defense, Defense Digital Service.
+// Released as open source under the MIT License.  See LICENSE file.
+//
+// =================================================================
```

Comment:
We really should make a pre-commit hook to add this to files. Then we can have it auto-add for us to new files.

Reply:
yep, not a blocker. Just something to keep in mind for the future.